crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -149,6 +152,20 @@ def cancel_allowed?
       ].include?(current_step) && !running?(:admin)
     end
 
+    def save_crowbar_backup(backup_location = "")
```

Comment:
it's just when you don't submit any parameter the path will be empty, isn't that wrong? why not just enforce the path by not giving a default?